examples/nunjucks/components/_preview.njk

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html class="antialiased text-gray-800">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+        <title>Preview Layout</title>
```

Comment:
value of title could be dynamic, e.g. we can use _self.title (or something) here

Reply:
I always thought this was useless, but it actually isn’t! It’s obviously the title of the preview page, how could I have missed that. Thanks for the hint, it’s fixed.